web/app/View/Composers/PageMockedComposer.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Wikijump\View\Composers;
+
+use Illuminate\View\View;
+use Faker;
+use Wikijump\Services\License\License;
+use Wikijump\Services\License\LicenseMapping;
+
+class PageMockedComposer
+{
+
+    private Faker\Generator $faker;
+
+    /** Create a new mocked page composer. */
+    public function __construct()
+    {
+        $this->faker = Faker\Factory::create();
+    }
+
+    /** Bind data to the view. */
+    public function compose(View $view)
+    {
+        $f = $this->faker;
+
+        $content = $this->generateContent();
+        $sidebar_content = $this->generateSidebarContent();
+        $navbar_items = $this->generateNavbarItems();
+        $license = $this->getRandomLicense();
+        $breadcrumbs = $this->generateBreadcrumbs();
+
+        $page_title = $f->streetName;
+        $revision = $f->randomNumber(3);
+        $timestamp = $f->dateTimeThisYear->getTimestamp();
+        $tags = $f->unique()->words(10);
+        $title = $page_title . ' | Wikijump';
```

Comment:
We shouldn't hardcode "Wikijump" anywhere:
```suggestion
        $title = $page_title . ' | ' . GlobalProperties::$SERVICE_NAME;
```

Reply:
This code is also incorrect because it should be using the site name. I suppose I could make it random. Also, a bunch of namespaces start with `Wikijump`, so like... we're deep in hardcoding that name.

Reply:
Well let me be more precise, in the code, the service name should be easily customizable. The code itself is obviously going to call itself "wikijump".